.travis.yml

```diff
@@ -59,6 +59,12 @@ matrix:
       - pip install -r cirq/contrib/contrib-requirements.txt
       - pip install -r dev_tools/conf/pip-list-dev-tools.txt
     script: check/performance
+
+  - os: osx
+    env: NAME=pytest (macOS)
+    python: "3.7.0"
+    language: generic      
+    script: ./dev_tools/continous_integration_macos.sh
```

Comment:
This should be the `install` step, and then the `script` step is `pytest`. You may need to use `source dev_tools/continous_integration_macos.sh` in order for it to work properly.

(Right now you're not actually running the tests!)